MapAnalyzer/MapData.py

```diff
@@ -503,11 +512,15 @@ def closest_towards_point(
                 >>> best_siege_spot = self.closest_towards_point(points=corners, target=enemy_army_position)
                 (57,120)
         """
-        if isinstance(points, list):
-            return points[self.closest_node_idx(node=target, nodes=points)]
-        else:
+        if isinstance(points[0], Point2):
+            # Converting to ndarray from List[Point2] is much slower
```

Comment:
I agree,  but i would like to have in this code comment some ball park of how slower it is,   
maybe a benchmark for a list of 100 points,  and a list of 500 points